doc/sphinxman/source/introduction.rst

```diff
@@ -472,114 +472,85 @@ For more details, see Tables :ref:`Energy <table:energy_gen>`,
-    +-------------------------+-----------+-----------+----------------------+-----------------------------+
-    | DMRG-SCF                | Y         | ---       |                      |                             |
-    +-------------------------+-----------+-----------+----------------------+-----------------------------+
-
-..    | %HF DBOC                | Y         | N         |
-    +-------------------------+-----------+-----------+
-    | %TCSCF                  | Y         | Y         |
-    +-------------------------+-----------+-----------+
-    | %CASSCF                 | Y         | Y         |
-    +-------------------------+-----------+-----------+
-    | %RASSCF                 | Y         | Y         |
-    +-------------------------+-----------+-----------+
-    | %RHF MP2-R12            | Y         | N         |
-    +-------------------------+-----------+-----------+
-    | %RAS-CI DBOC            | Y         | N         |
-    +-------------------------+-----------+-----------+
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | Method                  | Energy                             | Gradient                      | Parallelism                 | Variants               |
+    +                         +----------------+-------------------+--------------+----------------+                             +-----------+------------+
+    |                         | Reference      | Type              | Reference    | Type           |                             | OO [#f9]_ | FNO [#f1]_ |
+    +=========================+================+===================+==============+================+=============================+===========+============+
+    | HF                      | RHF/UHF/ROHF   | CONV/DF/CD        | RHF/UHF/ROHF | CONV/DF        | threaded                    |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | DFT                     | RKS/UKS        | CONV/DF/CD [#f7]_ | RKS/UKS      | DF [#f4]_      | threaded                    |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | EFP [#f5]_              | RHF            |                   | ---          | ---            |                             |           |            |
+    +-------------------------+----------------+-------------------+--------------+----------------+-----------------------------+-----------+------------+
+    | DCFT                    | UHF            | CONV              | UHF          | CONV           | partially threaded          |           |            |
```

Comment:
That was true when I finished with DCFT, before I turned to the "dark side" and developed MD methods.  Since then, the CCQC folks have done some really great work with the code.  @jturney and/or @ssh2 can hopefully provide an update reflecting these developments.